proto/osprey/rpc/labels/v1/service.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package osprey.rpc.labels.v1;
+
+// Mock proto file for deprecated labels functionality
+// These are marked as DEPRECATED in sync_action service
+// and will be removed once labels are replaced with verdicts
+
+message EntityKey {
```

Comment:
These seems to be the bare minimum to make the build pass without touching the Rust code. I'm add it for now just to make the build work. It will be removed in a separate PR when I remove calls to label service all together.